### DISCUSSION TITLE
Predicting probabilities using pyGAD

### DISCUSSION THREAD

**Discussion by p-suresh-kumar** [ahmedfgad/GeneticAlgorithmPython#29] [Ideas]
How to find predict probabilities of a classification model in the pyGAD.

or is there any function in pyGAD to find the ROC curve

**Comment by ahmedfgad** [comment]
PyGAD only optimizes the model and it has nothing to do with calculating metrics like accuracy or creating the ROC curve. 

After PyGAD trains the model, then you can use the correct and predicted outputs with other libraries like scikit-learn to calculate the metrics for you.

Please let me know if you have any other questions.

**Reply by p-suresh-kumar** [reply]
Dear GAD, 
Thank u v much for your response. 
i m getting predictions from your pyGAD only. can u show me how to get the score of best model using sklearn after model training over

**Reply by ahmedfgad** [reply]
You can compare the predictions to the correct outputs using the sklearn.metrics.accuracy_score() function. Please check this link: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html
